practice-app/test/test_home.py

```diff
@@ -1,43 +1,49 @@
 import unittest
-import pymongo
-import requests
+import datetime
+from api.home.home import *
+from werkzeug.exceptions import HTTPException
+from unittest.mock import patch
+import mock
+
+
+
+class MyTestCase(unittest.TestCase):
```

Comment:
MyTestCase doesn't feel like a good test class name. Might want to change it

Reply:
Fixed